doc/migrating/0.4-Migration.md

```diff
@@ -490,7 +512,7 @@ var grid = new (declare([ OnDemandGrid, Tree ]))({
 ```
 
 If you need to define the `getChildren` and `mayHaveChildren` methods yourself, it's worth looking at the source of the
-[`dstore/Tree`](https://github.com/SitePen/dstore/blob/master/Tree.js) module to see its usage of the `root` property.
+[`dstore/Tree`](https://github.com/SitePen/dstore/blob/master/src/Tree.js) module to see its usage of the `root` property.
```

Comment:
Good catch here, but I think I'm going to lock down this URL to a 1.x version or branch of dstore instead to prevent confusion (since master isn't exactly in a prime-time-ready state).